python/pythonlab/neighborhood/neighborhood/painter.py

```diff
@@ -259,8 +257,8 @@ def _send_signal(self, signal_key, detail=None):
     """
     detail = detail or {}
     detail['id'] = self.id
-    signal_message = NeighborhoodSignalMessage(SignalMessageType.PAINTER, signal_key.value, detail)
-    print(signal_message.get_formatted_message())
+    signal_message = NeighborhoodSignalMessage(SignalMessageType.NEIGHBORHOOD, signal_key, detail)
+    signal_message.send(self.world.context_type, NeighborhoodTracker._instance, is_boolean_message)
```

Comment:
When we use `NeighborhoodTracker._instance` does that only work if the neighborhood tracker has already been initialized elsewhere?

Reply:
Ahh yes, that's correct. 
We would assume it would be initialized by the `validation_protocol`, but I can update this so that we initialize here just in case.
Btw - I am passing the neighborhood tracker instance here bc when I imported `NeighborhoodTracker` in `neighborhood_signal_message`, I got a circular reference error.